src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -386,6 +386,12 @@ def link_content(text, css_class, icon)
       text
     end
   end
+
+  def sidebar_collapsed?
+    return true if cookies[:sidebar_collapsed].eql?('true')
```

Comment:
This method could be simplified to a one-liner like this:

```suggestion
    cookies[:sidebar_collapsed].eql?('true')
```